_unittest/test_utils.py

```diff
@@ -0,0 +1,70 @@
+"""Test utility functions of PyAEDT.
+"""
+
+import logging
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+import pytest
+
+from pyaedt import settings
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+SETTINGS_RELEASE_ON_EXCEPTION = settings.release_on_exception
+SETTINGS_ENABLE_ERROR_HANDLER = settings.enable_error_handler
+
+
+@pyaedt_function_handler(deprecated_arg="trigger_exception")
+def foo(trigger_exception=True):
+    """Some dummy function used for testing."""
+    if trigger_exception:
+        raise Exception("Dummy message.")
```

Comment:
Do we test the content of this message in our tests?
Just to be sure it is still working.

Reply:
Done